src/sentry/sentry_apps/tasks/sentry_apps.py

```
@@ -326,22 +326,29 @@ def process_resource_change_bound(
 def installation_webhook(installation_id: int, user_id: int, *args: Any, **kwargs: Any) -> None:
     from sentry.sentry_apps.installations import SentryAppInstallationNotifier
 
-    extra = {"installation_id": installation_id, "user_id": user_id}
-    try:
-        # we should send the webhook for pending installations on the install event in case that's part of the workflow
-        install = SentryAppInstallation.objects.get(id=installation_id)
-    except SentryAppInstallation.DoesNotExist:
-        logger.info("installation_webhook.missing_installation", extra=extra)
-        return
+    with SentryAppInteractionEvent(
+        operation_type=SentryAppInteractionType.PREPARE_WEBHOOK,
+        event_type="installation_webhook",
+    ).capture() as lifecycle:
+        lifecycle.add_extras({"installation_id": installation_id, "user_id": user_id})
 
-    user = user_service.get_user(user_id=user_id)
-    if not user:
-        logger.info("installation_webhook.missing_user", extra=extra)
-        return
+        try:
+            # we should send the webhook for pending installations on the install event in case that's part of the workflow
+            install = SentryAppInstallation.objects.get(id=installation_id)
+        except SentryAppInstallation.DoesNotExist:
+            error = SentryAppSentryError(message=SentryAppWebhookFailureReason.MISSING_INSTALLATION)
+            sentry_sdk.capture_exception(error)
```

a wee jank, but the retry decorator doesnt capture exceptions for ignored errors, so we need to capture and raise here

---

Should we keep the gcp logs? Or do you plan to add more logs for debugging in another PR?

---

When the errors cross the context manager bounds they should record a failure and log for us. Also jfyi that we can catch and capture the error in Sentry in the [retry decorator](https://github.com/getsentry/sentry/pull/86566). If it turns out these errors happen a lot and we need more visibility to debug we can come back and add logs (?)